cloudslang-cli/src/main/java/io/cloudslang/lang/cli/utils/MetadataHelperImpl.java

```diff
@@ -63,8 +69,28 @@ private Metadata cleanWhiteSpaces(Metadata metadata, String lineBreak) {
     }
 
     private void cleanMapWhiteSpaces(Map<String, String> map, String lineBreak) {
-        for (String key : map.keySet()) {
-            map.put(key, map.get(key).trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            map.put(entry.getKey(), entry.getValue().trim().replaceAll("[\\s]+" + lineBreak, lineBreak));
+        }
+    }
+
+    private static class CustomOrderMetadataRepresenter extends Representer {
+        @Override
+        protected Set<Property> getProperties(Class<? extends Object> type)
+                throws IntrospectionException {
+            Set<Property> result = new LinkedHashSet<>();
+            Set<Property> set = super.getProperties(type);
+            for (Property property : set) {
+                if (property.getType().equals(String.class)) {
+                    result.add(property);
+                }
+            }
+            for (Property property : set) {
```

Comment:
can you extract a generic method for the `for` or use only one loop and two `if`s?
what happens when the type is not `String` or `Map`? should those values be ignored?